plugins/codecs/ndlz/test_ndlz.c

```diff
@@ -71,17 +71,17 @@ static int test_ndlz_4(blosc2_schunk *schunk) {
 
     decompressed = blosc2_schunk_decompress_chunk(schunk, ci, data_in, chunksize);
     if (decompressed < 0) {
-      printf("Error decompressing chunk \n");
-      return -1;
+      BLOSC_TRACE_ERROR("Error decompressing chunk \n");
+      return BLOSC2_ERROR_FAILURE;
```

Comment:
It is not necessary to use the `BLOSC2_ERROR_FAILURE  / BLOSC_TRACE_ERROR ` mechanism in tests or executables in general.  This is more meant for the library itself, because it is designed to be used from other programs, and messing with standard I/O is not a good practice.